### PR TITLE
My Home: Add a Quick Start Session card.

### LINKED ISSUE #38998: Customer Home: Quick Start Sessions
## Start
Screen Shot 2020-02-26 at 11 27 58 AM

- Copy for card is 👍 good to go 
- Imagery for the card is 👍 good to go

- CTA for the card by default is set to a secondary button. When a user selects, it can go to:
`https://wordpress.com/me/concierge/[SITE]/book`

## When a user has a session scheduled
Screen Shot 2020-02-26 at 11 36 06 AM

- The copy that changes to reflect the session details is 👍 good to go

- The `View Details` CTA takes a user to `https://wordpress.com/me/concierge/[SITE]/book`

- The `Reschedule` link takes a user to `https://wordpress.com/me/concierge/[SITE]/36109/cancel`

### More info for card styling
In `/hosting-config`, there is a card for contacting Support. The most relevant part of it is how the avatars are generated, it uses this class:
`.support-card__avatars`

### Related work
There's WIP to make the scheduling flow show up in a dialog, so that the user isn't "taken away from" this page. That work shouldn't block this work. The net value users will get from seeing this outweighs whatever interaction pattern we choose.

### PR DESCRIPTION
**Pull request by kwight**
This PR adds a Quick Start Sessions card to My Home.

Not Booked | Booked
--- | ---
image | image

#### Testing instructions

* Go to `/home/:site` for a site with either an eCommerce or Business plan.
* Verify the "Book a session" button goes to `/me/concierge/:site/book`.
* On prod, take a test site with a Business plan and register for a Quick Start session at the above booking URL. (Be sure to comment that we're just testing. 🙃)
* Back on this branch for the Business test site, verify that the card title changes appropriately.
* Verify the "View details" button goes to `/me/concierge/:site/book`.
* Verify the "Reschedule" link goes to `/me/concierge/:site/:sessionId/cancel`.

Fixes #38998

### CODE REVIEW

**Comment by jancavan** [comment]
Not sure if this has been discussed already, but what happens to this section if they've already booked a session during signup/onboarding? Will it just disappear?

Just some ideas:

- If they've purchased this add-on, but haven't picked a time slot, maybe button can change to "Schedule a session"?
- If they've already picked a time slot, maybe it could be a date/time reminder of the upcoming session?

**Comment by gwwar** [comment]
> but what happens to this card if they've already booked a session during signup/onboarding? 

75371361-38cf5900-588c-11ea-9fb2-104ca37aebd7

See https://github.com/Automattic/wp-calypso/issues/38998

**Comment by jancavan** [comment]
@gwwar Sorry, I edited my comment :)

Another state would be - if a user has purchased this, but has not scheduled a session yet.

**Comment by gwwar** [comment]
> if a user has purchased this, but has not scheduled a session yet.

@jancavan is that not the first mock? I don't think we're upselling this to free plans until we have additional controls internally for helping deal with the influx.

**Comment by jancavan** [comment]
> I don't think we're upselling this to free plans until we have additional controls internally for helping deal with the influx.

Oh ok. I thought it was an upsell. I think paid plans get this upsell too in Signup.

File: client/my-sites/customer-home/cards/features/quick-start/index.js
Comment (a8ci18n) [inline client/my-sites/customer-home/cards/features/quick-start/index.js:41]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Your scheduled Quick Start session details are:' )` *ES Score: 7*

File: client/state/reducer.js
Comment (kwight) [inline client/state/reducer.js:26]: These changes were needed for the Concierge query component to populate state with the API response (without it, the `getConciergeNextAppointment` selector was always `null`). Which makes sense – I've just never seen a set of reducers _not_ be in the root reducer, and I'm surprised that, like, all of Concierge isn't broken without it. Anyone see what's going on (or maybe my limited Redux knowledge has me not seeing something)? ¯\_(ツ)_/¯

Comment (gwwar) [reply client/state/reducer.js:26]: Which routes? It could be that it didn't work before

Comment (gwwar) [reply client/state/reducer.js:26]: Ah found it https://github.com/Automattic/wp-calypso/pull/30480 so we probably don't want to re-add here

Comment (kwight) [reply client/state/reducer.js:26]: Ah, I guess this is related; so the Concierge reducers are only being registered for select routes, and I've gone and shoved them in for all routes?.. This shows up when proceeding to a concierge route in this PR:

Screen Shot 2020-04-14 at 4 45 29 PM

Comment (gwwar) [reply client/state/reducer.js:26]: Yeah I think we were trying to slim down the main reducer bundle, it'd be worth chatting with @Automattic/team-calypso to see if it makes sense to add back. (As we need this in the MySites default page).

File: client/my-sites/customer-home/cards/features/quick-start/index.js
Comment (a8ci18n) [inline client/my-sites/customer-home/cards/features/quick-start/index.js:53]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Reschedule session' )` *ES Score: 8*

File: client/state/reducer.js
Comment (kwight) [reply client/state/reducer.js:26]: @gwwar ah, didn't see your comment, thanks! That's interesting. So we would do the same async load thing as that PR, but only if the given site has a plan qualifying for Quick Start sessions?

Comment (gwwar) [reply client/state/reducer.js:26]: > but only if the given site has a plan qualifying for Quick Start sessions?

Yeah, let's give it a try. If we open this card up to everyone on a free plan, it probably would make more sense to add back to the main reducer since this is the default section we land on.

File: client/my-sites/customer-home/locations/secondary/index.jsx
Comment (kwight) [inline client/my-sites/customer-home/locations/secondary/index.jsx:29]: https://github.com/Automattic/wp-calypso/pull/40993/files#r406412366

File: client/state/reducer.js
Comment (kwight) [reply client/state/reducer.js]: I got this working in 5aea6d8437, but it feels really heavy-handed and... special compared to always having it in the root reducer like so much else. I'm not sure how to measure the performance impact though, done this way. @gwwar @jsnajdr any preferences either way?

Comment (gwwar) [reply client/state/reducer.js]: @jsnajdr's on leave. Overall, @Automattic/team-calypso @griffbrad I think we'll eventually want to add this back to the main bundle when the card is visible to folks on all plans.

For now I could go either way on adding it back for simplicity or trying to keep another async load. @kwight for the perf impact I suppose we could profile locally trying async vs the reducer bundle with caches disabled.

**Comment by kwight** [comment]
I've updated instructions for testing with D41945-code (and I've removed the hard-coded instance of this component).

File: client/my-sites/customer-home/cards/features/quick-start/style.scss
Comment (mmtr) [inline client/my-sites/customer-home/cards/features/quick-start/style.scss:1]: I think this refers to the wrong card.
```suggestion
.quick-start p {
```

File: client/my-sites/customer-home/controller.jsx
Comment (mmtr) [inline client/my-sites/customer-home/controller.jsx:22]: Can we maybe use `isBusinessPlan` and `isEcommercePlan`?

https://github.com/Automattic/wp-calypso/blob/c78781876f716e094f9628c5c514a469f32508f6/client/lib/plans/index.js#L217-L223

**Review by mmtr — COMMENTED** [review]
This is looking good, @kwight! I know this is still in progress, so not sure what are the things that can actually be tested, so I'll share my finds just in case. Please ignore everything if that's the expected result at this stage 🙂 

- ✅ I get the card on eligible sites, ❌but the "Book a session" doesn't link to `/me/concierge/:site/book`.
- ✅If I have a session booked, the cards the card title changes appropriately, ❌but seems styles need to be adjusted a little bit.
Screen Shot 2020-04-17 at 12 19 36

- ✅If I have a session booked, the card shows up when switching to an ineligible site (since the session is attached to my user, not my site).
- ❌Either clicking on "View details" or "Reschedule" results in an error:
Screen Shot 2020-04-17 at 12 21 07

File: client/my-sites/customer-home/controller.jsx
Comment (gwwar) [inline client/my-sites/customer-home/controller.jsx:22]: 🤔 I wonder if thisoptimization is worth it or not, we could also gate based on if the card is returned by the layout API. 

For simplicity sake, I'd probably be fine with dropping the check here and adding to the main reducer if Team Calypso doesn't object.

**Comment by gwwar** [comment]
Here's a snapshot of the WIP, looking good so far Kirk

Screen Shot 2020-04-17 at 2 25 23 PM

File: client/state/reducer.js
Comment (kwight) [reply client/state/reducer.js]: I'll eave it async for now, and we can worry about moving to the root reducer if/when we open the card to all users.

File: client/my-sites/customer-home/controller.jsx
Comment (kwight) [reply client/my-sites/customer-home/controller.jsx]: Oooh, yes, good idea 👍 9f1ab155c1

Comment (kwight) [reply client/my-sites/customer-home/controller.jsx]: Ah, OK, I can do that next week if we don't hear from anyone.

It _would_ fix another small issue I just found with this "check plans" method – but I'm uncertain it's an issue or not. The server-side logic for the card is "show if the current site has a Business or eCommerce plan, _or_ if the current user already has an appointment booked for _any_ site". BY moving the reducer to the root, we don't have to worry about the second case (which isn't covered in the current implementation).

But is that what we want? should Home show a user's appointment for _any_ site, or just the site being displayed? I could see user confusion both ways: "I thought I booked one of those appointments!" or "I don't have a plan on this site?..".

**Comment by kwight** [comment]
> but seems styles need to be adjusted a little bit

@mmtr Thanks! yes, styling still to come (I was last working on server-side tests, and sorting the date formatting in the card).

> Either clicking on "View details" or "Reschedule" results in an error

I'm having problems with this too, and being constantly asked for 2FA auth again, but I think I was getting it on `master` too (but not prod).

**Comment by kwight** [comment]
> but the "Book a session" doesn't link to /me/concierge/:site/book.

Oops, yes, this went missing at some point: da7c657a45

File: client/my-sites/customer-home/controller.jsx
Comment (kwight) [reply client/my-sites/customer-home/controller.jsx]: We're going to only show the "existing session" version if it's for the current site.  I'll update that server-side, and add the reducer to the root reducer (as we haven't heard anything to the contrary).

File: client/my-sites/customer-home/cards/features/quick-start/index.js
Comment (kwight) [inline client/my-sites/customer-home/cards/features/quick-start/index.js:85]: This needs to also compare blog ID now (it should only be `true` if there is a session, _and_ it's for the current blog being viewed.

**Comment by kwight** [comment]
We'll hold off on merging this until we've verified that we have the support coverage in place to handle the increased Quick Start interest. See pau2Xa-1bk-p2.

File: client/my-sites/customer-home/controller.jsx
Comment (kwight) [reply client/my-sites/customer-home/controller.jsx]: cb43e05

File: client/my-sites/customer-home/cards/features/quick-start/index.js
Comment (kwight) [reply client/my-sites/customer-home/cards/features/quick-start/index.js:85]: Hm, there _is_ no blog ID associated with the appointment object. I'll have to dig into how the concierge pages themselves handle similar checks.

**Comment by kwight** [comment]
This should be good for a final look – then we can push it aside until given the go-ahead to launch it.

Does anyone else have problems going to the booking URL (`/me/concierge/:site/book`)? I constantly get prompted for 2FA verification, and once it passes, the page just sits on the placeholder (from what I can tell, it's not getting user settings?). I get the same behaviour on `master`, but not on prod, so... I guess we're fine? Curious if anyone else is seeing it.

File: client/my-sites/customer-home/cards/features/quick-start/index.js
Comment (mmtr) [inline client/my-sites/customer-home/cards/features/quick-start/index.js:63]: Probably this href is the reason why both View details and Reschedule redirect to the same page for me.

**Review by mmtr — COMMENTED** [review]
Cool, this is pretty close! I think we need to also implement a different card version for the experimental Home layout, but not a blocker here.

According to https://github.com/Automattic/wp-calypso/issues/38998#issuecomment-619254624, I was expecting to view my scheduled session when visiting My Home of any of my sites, but I can see it only from Business or eCommerce sites.

I also noted that after cancelling a scheduled session, if I go back to My Home, the card still shows the (now cancelled) session. I had to reload the page in order to view the "Book a session" button again.

> Verify the "View details" button goes to /me/concierge/:site/book.
> Verify the "Reschedule" link goes to /me/concierge/:site/:sessionId/cancel.

Both redirect to `/me/concierge/:site/book` for me.

> Does anyone else have problems going to the booking URL (/me/concierge/:site/book)? I constantly get prompted for 2FA verification, and once it passes, the page just sits on the placeholder

I think we have some kind of 2FA weirdness when testing locally. I tested using the calypso.live branch and didn't have any issue at all, so I think we're safe 👍

File: client/my-sites/customer-home/cards/features/quick-start/index.js
Comment (gwwar) [reply client/my-sites/customer-home/cards/features/quick-start/index.js:63]: +1 on both links taking me to /book

**Comment by gwwar** [comment]
Pretty close Kirk! Some things to consider before landing the client piece:
- [ ] branch needs a rebase 
- [ ] Let's fix the reschedule link

Things to think about for follow up (@jancavan @kwight ):
- After scheduling/rescheduling, navigation back to My Home is a little awkward on a multi-site user
- Should we have a hint on other sites when a session is scheduled? Eg I have Site A and Site B, with a session scheduled for B. When looking at A for My Home. 🤔Maybe a user notification would be better.

**Comment by jancavan** [comment]
> Should we have a hint on other sites when a session is scheduled? Eg I have Site A and Site B, with a session scheduled for B. When looking at A for My Home. 🤔Maybe a user notification would be better.

Wasn't the plan to just show the concierge card on all sites the user owns (since they really could only book one even if had have multiple sites)? Although some indication as to which site it is for would be great.

If the user intends to book a session for Site A, but mistakenly books it for Site B, they don't have to cancel and reschedule for the right site, right? I'm assuming they can just move forward with the call, but discuss Site A instead?

**Comment by gwwar** [comment]
> If the user intends to book a session for Site A, but mistakenly books it for Site B, they don't have to cancel and reschedule for the right site, right? I'm assuming they can just move forward with the call, but discuss Site A instead?

Not sure, we'll need to ask folks

**Comment by jancavan** [comment]
I think displaying the same card all across the user's sites makes sense if they can only book a single session anyway, but let me ask.

**Comment by jancavan** [comment]
> If the user intends to book a session for Site A, but mistakenly books it for Site B, they don't have to cancel and reschedule for the right site, right? I'm assuming they can just move forward with the call, but discuss Site A instead?

This is correct. If there has been a mistake, there is no need to reschedule for the correct site. The HE can just mark the correct site after the session has completed.

**Comment by gwwar** [comment]
@kwight @jancavan Chatting with @wynwin we'll want to start with showing this card for **only folks who already have a Quick Start Session scheduled**.

Folks will let us know when we're ready to increase visibility to more customers.

**Comment by kwight** [comment]
I've updated this to only display if there's an existing booking, pairing it with D42780-code (experimental layout only). It's not working yet though (I think the query component might be involved).

File: client/my-sites/customer-home/cards/features/quick-start/index.js
Comment (a8ci18n) [inline client/my-sites/customer-home/cards/features/quick-start/index.js:48]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Your scheduled Quick Start session details are:' )` *ES Score: 7*

File: client/my-sites/customer-home/cards/features/quick-start/index.js
Comment (a8ci18n) [inline client/my-sites/customer-home/cards/features/quick-start/index.js:72]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Reschedule session' )` *ES Score: 8*

**Comment by gwwar** [comment]
> I've updated this to only display if there's an existing booking, pairing it with D42780-code (experimental layout only). It's not working yet though (I think the query component might be involved).

Let's move that display check to the API. We'll also be able to show some other card instead if we filter on the server.

**Comment by gwwar** [comment]
I suppose we can drop https://github.com/Automattic/wp-calypso/pull/41112/commits/7fe403d880fd519faaa8c2941f285c745d105af2 if the API side is handling this.

**Comment by gwwar** [comment]
Really close @kwight !
- [x] API now only tells the client to render this when a session is booked. This will show up on My Home any sites that the user can manage.
- [ ] @jancavan any suggested copy changes since sessions are booked per user, but with a focus for a specific site?
- [ ] @kwight "Reschedule" still redirects to /book instead of /cancel or /reschedule

File: client/my-sites/customer-home/cards/features/quick-start/index.js
Comment (kwight) [reply client/my-sites/customer-home/cards/features/quick-start/index.js:92]: Don't need to worry about this anymore.

**Comment by jancavan** [comment]
> any suggested copy changes since sessions are booked per user, but with a focus for a specific site?

What about just adding the site URL like (quick mockup):

Screen Shot 2020-05-01 at 11 06 22 AM

I'd also add a colon after "Your scheduled Quick Start support session:" since it introduces the list of scheduling details below it.

**Comment by danhauk** [comment]
@jancavan That would be my suggestion as well. I like your mockup there.

**Comment by gwwar** [comment]
Behavior notes since the `/me/concierge` is a bit funky:

* Only **Business Plan owners** are allowed to book. And concierge shows an incorrect need to upgrade message if another admin tries booking a session for that site.

Screen Shot 2020-05-01 at 11 21 21 AM

* Plans however are tied to sites/users.
* A user may only book one session, even if they have sites with multiple plans. Visiting `/me/concierge/siteB/book` with a session already booked for siteA will incorrectly show that they have a session for siteB.

* Based on the one session max per user, I think it's fine if we display the card on other My Home sites, but we'll likely run into some complexity displaying this correctly. @kwight feel free to limit the API to only that scheduled site for a first implementation if this ends up being too much work.

**Comment by kwight** [comment]
> Behavior notes since the /me/concierge is a bit funky

Ha, yes, I just came to report all of this. There really is no concept of a session belonging to a site; there is no session information that ties it to a blog, only a user, which makes displaying the session's site in the card impossible at this point (without adding more plumbing to the Concierge system).

>  @kwight feel free to limit the API to only that scheduled site for a first implementation

Again, same thing; there's no way to identify this sever-side either. We can either:
* show the card for all sites if a session is booked, or
* show the card for all business sites if a session is booked,
... and neither would be able to display an associated site.

I think the best way forward at this point would be showing the card on business sites if a session is booked. Most users will only have one business plan, and since a session can discuss anything the user wants (it's not like HEs limit the conversation to a certain business plan), that card would not be surprising if a session is booked. It's a starting point at least?

**Comment by gwwar** [comment]
> Again, same thing; there's no way to identify this server-side either. 

🦆wat? 🙃

> I think the best way forward at this point would be showing the card on business sites

Let's swap that to a plan owner check on the API I think? Folks won't be able to book on a site they don't own the business plan of.

**Comment by kwight** [comment]
> I'd also add a colon after "Your scheduled Quick Start support session:" since it introduces the list of scheduling details below it.

👍 89b7e1d793d0a79843b2e043126df46af3b148d2

File: client/my-sites/customer-home/cards/features/quick-start/index.js
Comment (a8ci18n) [inline client/my-sites/customer-home/cards/features/quick-start/index.js:45]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Your scheduled Quick Start session details are:' )` *ES Score: 7*

**Comment by kwight** [comment]
> Again, same thing; there's no way to identify this server-side either.

I lied! There _is_ a `site_id` property on the Concierge appointment objects – so we _can_ determine server-side if a session is for a certain site. But that site ID does not make it to the client; here's what's happening.

When My Home and the Quick Start card loads, the `nextSession` prop will be `null`, triggering the `QueryConciergeInitial` query component, which takes a site ID. This makes a call to `https://public-api.wordpress.com/wpcom/v2/concierge/initial?_envelope=1&site_id=:site_id`. One of the first things the endpoint does is call `get_eligible_schedule_id` with the site ID; if the site ID isn't found in existing sessions, nor is it associated with valid plans, then the endpoint returns a null `next_appointment`. So if we're on a My Home page for a non-qualifying site, we'll never see any existing plan for other sites.

We have a few options:
* modify the query component (or make another) so that it _doesn't_ rely on a site ID,
* glob the site ID of any sessions found server-side into the `/home/layout` response, so that it can be used for the query component,
* or stick with just displaying the card on sites that have qualifying plans, and add the session's site ID into the `/concierge/initial/` response so that the card can identify which site the session is for.

**Comment by gwwar** [comment]
@kwight Since there seems to be a lot of work in general for making this feature make more sense, let's maybe stick with the simplest option and only have the layout API return this card if someone has a booked session with that site_id on the Concierge appointment object

**Comment by kwight** [comment]
> let's maybe stick with the simplest option and only have the layout API return this card if someone has a booked session with that site_id on the Concierge appointment object

D42832-code

File: client/my-sites/customer-home/cards/features/quick-start/index.js
Comment (a8ci18n) [inline client/my-sites/customer-home/cards/features/quick-start/index.js:47]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Your scheduled Quick Start session details are:' )` *ES Score: 7*

File: client/my-sites/customer-home/cards/features/quick-start/index.js
Comment (a8ci18n) [inline client/my-sites/customer-home/cards/features/quick-start/index.js:74]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Reschedule session' )` *ES Score: 8*

File: client/my-sites/customer-home/cards/features/quick-start/index.js
Comment (a8ci18n) [inline client/my-sites/customer-home/cards/features/quick-start/index.js:79]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Reschedule session' )` *ES Score: 8*

**Comment by kwight** [comment]
I played with a couple of different versions for the loading status. I started with a spinner, but it was weird trying to get the height of the card to not jump around as markup was changing (I could use a bunch of magic `height` values at different breakpoints, but that felt wrong).



I ultimately settled on changing as little as possible for the markup (so that card height never varies), and had the content and buttons themselves adapt to the loading state.



(Not sure why my gif software decided everything will be b&w for the rest of the day, but voila.)

File: client/my-sites/customer-home/cards/features/quick-start/index.js
Comment (gwwar) [inline client/my-sites/customer-home/cards/features/quick-start/index.js:99]: This needs to be `site_slug` or tracks will reject the event

File: client/my-sites/customer-home/cards/features/quick-start/index.js
Comment (gwwar) [inline client/my-sites/customer-home/cards/features/quick-start/index.js:111]: This needs to be `site_slug` or tracks will reject the event

**Review by gwwar — APPROVED** [review]
Awesome @kwight @jancavan 💖! I verified that:
* The card now only displays for an existing scheduled session for a given site
* We don't see this accidentally on other home pages
* Both links on the card work as expected

This is good to 🚢 once we fix analytics events

**Comment by gwwar** [comment]
Btw for loaders, @kwight I think we usually just use the pulsing bar over other options, but this is something we can iterate on if folks had feedback in follow ups.

**Comment by kwight** [comment]
> This is good to 🚢 once we fix analytics events

Done in 9d7b5d3 (well spotted, thanks!)

File: client/my-sites/customer-home/cards/features/quick-start/index.js
Comment (a8ci18n) [inline client/my-sites/customer-home/cards/features/quick-start/index.js:79]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Reschedule session' )` *ES Score: 8*

**Comment by kwight** [comment]
> I think we usually just use the pulsing bar over other options

Good point, added 👍

File: client/my-sites/customer-home/cards/features/quick-start/index.js
Comment (a8ci18n) [inline client/my-sites/customer-home/cards/features/quick-start/index.js:83]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Reschedule session' )` *ES Score: 8*

File: client/my-sites/customer-home/cards/features/quick-start/index.js
Comment (kwight) [reply client/my-sites/customer-home/cards/features/quick-start/index.js]: Ooh, well spotted. I've updated to use the site ID, which is more consistent with other cards anyways. 9d7b5d350424aeeb474e3dcb128e900a0e4e0b66

File: client/my-sites/customer-home/cards/features/quick-start/index.js
Comment (kwight) [reply client/my-sites/customer-home/cards/features/quick-start/index.js]: 9d7b5d350424aeeb474e3dcb128e900a0e4e0b66

File: client/my-sites/customer-home/cards/features/quick-start/index.js
Comment (a8ci18n) [inline client/my-sites/customer-home/cards/features/quick-start/index.js:83]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Reschedule session' )` *ES Score: 8*

**Pull request merged**
